build.assets/docker/base.dockerfile

```diff
@@ -55,6 +55,36 @@ RUN export DEBIAN_FRONTEND=noninteractive && set -ex && \
         && apt-get -y autoclean && apt-get -y clean && apt-get autoremove \
         && rm -rf /var/lib/apt/lists/*;
 
+# We need to use a newer version of iptables than debian has available
+# not ideal, but it's easier to run `make install` if we run this inline instead of a multi-stage build
+RUN export DEBIAN_FRONTEND=noninteractive && set -ex \
+        && apt-get update \
+        && apt-get install -q -y --allow-downgrades --no-install-recommends \
+        git \
+        autoconf \
+        libtool \
+        automake \
+        pkg-config \
+        libmnl-dev \
+        make \
+        && mkdir /tmp/iptables.build \
+        && git clone git://git.netfilter.org/iptables.git --branch ${IPTABLES_VER} --single-branch /tmp/iptables.build \
```

Comment:
I'm curious why you chose to git clone it instead of just curling the tarball with the sources.

Reply:
No reason in particular, I don't see one approach vs the other having significant advantages. I suppose curl has a small advantage that we can set a bunch of retries for temporary failures. But at the same time I get corrupted curl downloads on my gce build nodes that I haven't been able to figure out. If we were using magnet, download the sources would be probably be preferable as that's much more setup for caching of downloaded artifacts, assuming the server is able to return an etag.